usmqe/api/tendrlapi/common.py

```diff
@@ -174,3 +174,18 @@ def ping(self, asserts_in=None):
         self.print_req_info(response)
         self.check_response(response, asserts_in)
         return response.json()
+
+    def flows(self, asserts_in=None):
+        """ Ping REST API
+        Name:        "flows",
+        Method:      "GET",
+        Pattern:     "Flows",
+        """
+        pattern = "Flows"
+        response = requests.get(
+            pytest.config.getini("usm_api_url") + pattern,
+            auth=self._auth,)
+        self.print_req_info(response)
+        self.check_response(response, asserts_in)
+        # TODO: some minimal validation of flows response?
```

Comment:
Flows is dynamically generated from definitions files so there could be some complicated check if it generates flows properly but I would just add check for valid json if any.